libsolidity/interface/CompilerStack.cpp

```diff
@@ -700,21 +751,17 @@ bool CompilerStack::compile(State _stopAfter)
 					}
 					catch (UnimplementedFeatureError const& _unimplementedError)
 					{
-						if (
-							SourceLocation const* sourceLocation =
-							boost::get_error_info<langutil::errinfo_sourceLocation>(_unimplementedError)
-						)
+						if (SourceLocation const* sourceLocation =
+								boost::get_error_info<langutil::errinfo_sourceLocation>(_unimplementedError))
```

Comment:
Wrapping/indentation still seems off here.
```suggestion
						if (
							SourceLocation const* sourceLocation =
								boost::get_error_info<langutil::errinfo_sourceLocation>(_unimplementedError)
						)
```

Also in the `error()` call below in the diff.